.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,50 +1,36 @@
-<!--- Before submitting your pull request, --->
-<!--- please complete as much as practicable of the following checklist: --->
+<!--- Type an x in the box [x] before submitting --->
 
-### Pull Request Checklist
+### Submission Checklist
 
-* [ ] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
-* [ ] Based this PR on the latest version of the correct branch (master or 3.x)
-* [ ] Followed [PEP 8](https://www.python.org/dev/peps/pep-0008/) code style
-* [ ] Ensured this pull request hasn't eliminated unrelated blank lines/spaces,
-      modified the ``spyder/defaults`` directory, or added new icons/assets
-* [ ] Wrote at least one-line docstrings following PEP 257for any new functions
-* [ ] Added at least one unit test covering the changes, if at all possible
-* [ ] Described the changes and the motivation for them below
-* [ ] Noted what issue(s) this pull request resolves, creating one if needed
-* [ ] Included a screenshot, if this PR makes any visible changes to the UI
+<!--- I have read the Contributing Guidelines:                         --->
```

Comment:
Since there is not checkbox anymore (which I think is a very good thing!), I would change this to something like that:

"Please read the Contributing Guidelines (https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md) and follow the PEP 8, PEP 257 and Spyder's code style: https://github.com/spyder-ide/spyder/wiki/Dev:-Coding-Style"

Reply:
Yep :+1: